packages/locale/lang/my.ts

```diff
@@ -0,0 +1,184 @@
+      yearTablePrompt:
+        'arrow keys နှင့် enter ကိုအသုံးပြုပြီး နှစ် ကိုရွေးချယ်ပါ',
+      selectedDate: 'ရွေးချယ်ထားတဲ့ ရက်စွဲ',
+      selectDate: 'ရက်စွဲကို ရွေးချယ်ပါ',
+      selectTime: 'အချိန်ကို ရွေးချယ်ပါ',
+      startDate: 'စတင်မည့်ရက်စွဲ',
+      startTime: 'စတင်မည့်အချိန်',
+      endDate: 'ကုန်ဆုံးမည့်ရက်စွဲ',
+      endTime: 'ကုန်ဆုံးမည့်အချိန်',
+      prevYear: 'ယခင်နှစ်',
+      nextYear: 'နောက်နှစ်',
+      prevMonth: 'ယခင်လ',
+      nextMonth: 'နောက်လ',
+      year: '',
+      month1: 'ဇန်နဝါရီ',
+      month2: 'ဖေဖော်ဝါရီ',
+      month3: 'မတ်',
+      month4: 'ဧပြီ',
+      month5: 'မေ',
+      month6: 'ဇွန်',
+      month7: 'ဇူလိုင်',
+      month8: 'သြဂုတ်',
+      month9: 'စက်တင်ဘာ',
+      month10: 'အောက်တိုဘာ',
+      month11: 'နိုဝင်ဘာ',
+      month12: 'ဒီဇင်ဘာ',
+      week: 'ရက်သတ္တပတ်',
+      weeks: {
+        sun: 'နွေ',
+        mon: 'လာ',
+        tue: 'ဂါ',
+        wed: 'ဟူး',
+        thu: 'ကြာ',
+        fri: 'သော',
+        sat: 'နေ',
+      },
+      weeksFull: {
+        sun: 'တနင်္ဂနွေ',
+        mon: 'တနင်္လာ',
+        tue: 'အင်္ဂါ',
+        wed: 'ဗုဒ္ဓဟူး',
+        thu: 'ကြာသပတေး',
+        fri: 'သောကြာ',
+        sat: 'စနေ',
+      },
+      months: {
+        jan: 'ဇန်',
+        feb: 'ဖေ',
+        mar: 'မတ်',
+        apr: 'ပြီ',
+        may: 'မေ',
+        jun: 'ဇွန်',
+        jul: 'လိုင်',
+        aug: 'ဩ',
+        sep: 'စက်',
+        oct: 'အောက်',
+        nov: 'နို',
+        dec: 'ဒီ',
+      },
+    },
+    inputNumber: {
+      decrease: 'အရေအတွက်လျှော့ချ',
+      increase: 'အရေအတွက်တိုး',
+    },
+    select: {
+      loading: 'ဝန်တင်နေသည်',
+      noMatch: 'ကိုက်ညီသောဒေတာမရှိပါ',
+      noData: 'ဒေတာမရှိပါ',
+      placeholder: 'ရွေးပါ',
+    },
+    mention: {
+      loading: 'ဝန်တင်နေသည်',
+    },
+    dropdown: {
+      toggleDropdown: 'Dropdown ဖွင့်/ပိတ်',
+    },
+    cascader: {
+      noMatch: 'ကိုက်ညီသောဒေတာမရှိပါ',
+      loading: 'ဝန်တင်နေသည်',
+      placeholder: 'ရွေးပါ',
+      noData: 'ဒေတာမရှိပါ',
+    },
+    pagination: {
+      goto: 'သွားမယ်',
+      pagesize: '/စာမျက်နှာ',
+      total: 'စုစုပေါင်း {total}',
+      pageClassifier: '',
+      page: 'စာမျက်နှာ',
+      prev: 'ရှေ့စာမျက်နှာသို့',
+      next: 'နောက်စာမျက်နှာသို့',
+      currentPage: 'စာမျက်နှာ {pager}',
+      prevPages: 'ရှေ့စာမျက်နှာ {pager} သို့',
+      nextPages: 'နောက်စာမျက်နှာ {pager} သို့',
+      deprecationWarning:
+        'ကန့်ကွက်ထားသော အသုံးပြုမှုများကို တွေ့ရှိပါတယ်။ အသေးစိတ်အချက်အလက်များကို el-pagination ရဲ့စာရွက်စာတမ်းတွင် ဖတ်ရှုပါ။',
+    },
+    dialog: {
+      close: 'ဤဒိုင်ယာလော့ဂ်ကို ပိတ်ပါ',
+    },
+    drawer: {
+      close: 'ဤဒိုင်ယာလော့ဂ်ကို ပိတ်ပါ',
+    },
+    messagebox: {
+      title: 'မက်ဆေ့ချ်',
+      confirm: 'အိုကေ',
+      cancel: 'ပယ်ဖျက်',
+      error: 'တရားမဝင်ထည့်သွင်းမှု',
+      close: 'ဤဒိုင်ယာလော့ဂ်ကို ပိတ်ပါ',
+    },
+    upload: {
+      deleteTip: 'ဖယ်ရှားရန် delete ကိုနှိပ်ပါ',
```

Comment:
Does `delete` need to be translated?

Reply:
@btea Thank you for your very quick and detailed reviews! I’ve also updated this translation. Let me know if you need any more adjustments!